### PR TITLE
added torchio recipe

### PR DESCRIPTION
**Pull request by sarthakpati**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sarthakpati** [comment]
Tagging @conda-forge/help-python for merge

File: recipes/torchio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{% set name = "torchio" %}
+{% set version = "0.18.29" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/torchio-{{ version }}.tar.gz
```
Comment (dopplershift) [inline recipes/torchio/meta.yaml:10]: There's one last "torchio" here that should be `{{ name }}`

File: recipes/torchio/LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (dopplershift) [inline recipes/torchio/LICENSE:1]: This LICENSE file is already included in the PyPI tarball, so go ahead and remove this file from the PR.

**Review by dopplershift — CHANGES_REQUESTED** [review]
A couple minor things and this will be good to go.

File: recipes/torchio/meta.yaml (hunk 1 above)
Comment (sarthakpati) [reply recipes/torchio/meta.yaml]: Done!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**